lib/screens/album_details/album_details_screen.dart

```diff
@@ -26,8 +27,47 @@ class AlbumDetailsScreen extends StatelessWidget {
           ),
         ),
       ),
-      body: Center(
-        child: Text('Details for album id: ${album?.id}'),
+      body: Container(
+        height: MediaQuery.of(context).size.height,
+        width: double.infinity,
+        padding: const EdgeInsets.only(top: 25, left: 25, right: 25),
+        child: Column(
+          children: <Widget>[
+            CircleAvatar(
+              backgroundColor: Theme.of(context).primaryColor,
+              radius: 40,
+              child: Text(
+                '${album!.title![0].toUpperCase()}',
+                style: Theme.of(context).textTheme.headline3,
+              ),
+            ),
+            Padding(
+              padding: const EdgeInsets.symmetric(vertical: 25),
+              child: Center(
+                child: Column(
+                  children: <Widget>[
+                    Text(
+                      overflow: TextOverflow.ellipsis,
+                      '${album!.title}',
+                      style: Theme.of(context).textTheme.headline5,
+                    ),
+                    SizedBox(
+                      height: 5,
+                    ),
+                    Text(
+                      'Album with id: ${album?.id}',
+                      style: Theme.of(context).textTheme.bodyText1,
+                    ),
+                  ],
+                ),
+              ),
+            ),
+            Divider(
```

Comment:
@AndreiBorzaRodeapps what about creating a new widget for this? AppDivider maybe?